use-platform/getting-started-for-the-impatient.rst

```diff
@@ -227,30 +216,23 @@ The default ``project.make`` file for a Drupal 7 installation looks like this:
   projects[platform][version] = 1.3
   projects[platform][subdir] = contrib
 
-If you are building with Drush Make, the proper place for your file is ``shared/settings.local.php``. The `Platform.sh CLI <https://github.com/platformsh/platformsh-cli>`_ will have created this file for you when you ran the platform get command.
-
-.. note::
-  If there is no shared/settings.local.php file, create one following the `example found here <https://github.com/platformsh/platformsh-cli/blob/master/resources/drupal/settings.local.php>`_, and re-run platform build.)
-
-When using Drush Make files, the ``platform build`` command will generate a `sites/default/settings.php` file with each build of your application. The `shared/settings.local.php` file will also be symlinked into the `www/sites/default` directory, where the generated settings.php can include it.
-
 Install Profile build mode
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If your project contains a profile file: ``*.profile``, the Platform.sh CLI builds your project in profile mode. This is similar to what Drupal.org does to build distributions. Everything you have in your repository will be copied to your ``profile/[name]`` folder.
+If your project contains a profile file: ``*.profile``, the Platform.sh CLI builds your project in profile mode. This is similar to what Drupal.org does to build distributions. Everything you have in your repository will be copied to your ``profiles/[name]`` folder.
 
 .. note::
   It is a mistake to mix Vanilla mode with other modes. If you've copied all of the Drupal core files into your repository then you need to make sure you don't have any ``*.make` or ``*.profile`` files.
 
-Connect to your local database
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Your local database credentials will be put in a ``settings.local.php`` file. Where this file is stored depends on what build mode you are using for Drupal.
-
 Database credentials
 ~~~~~~~~~~~~~~~~~~~~
 
-Whether your ``settings.local.php`` file is in `repository/sites/default/settings.local.php` (Vanilla mode) or `shared/settings.local.php` (Drush Make mode), you need to add your local database credentials.
+You need to add your local database credentials to a ``settings.local.php`` file.
+
+This will be stored in ``sites/default/settings.local.php`` in your local Drupal site. However, if you have used the CLI to build your project, then it's better to use ``shared/settings.local.php`` (inside the project root). The CLI will have created this file for you, when you ran the ``platform get`` or ``platform build`` command.
```

Comment:
FWIW - this is a bit unnecessary, as the CLI symlinks

`builds/<build>/sites/default/settings.local.php -> ../../../../shared/settings.local.php`